pkg/splunk/enterprise/clustermaster_test.go

```diff
@@ -61,6 +61,11 @@ func TestApplyClusterMaster(t *testing.T) {
 			Name:      "stack1",
 			Namespace: "test",
 		},
+		Spec: enterprisev1.ClusterMasterSpec{
+			CommonSplunkSpec: enterprisev1.CommonSplunkSpec{
+				Mock: true,
+			},
+		},
```

Comment:
Nit: For now it's OK. But I think a better way to handle this is by indicating a particular flag, other than the Spec, say, for example, a config map entry